README.md

```diff
@@ -1,4 +1,4 @@
-# KNXnetIP [![pipeline status](https://gitlab.uptime.dk/troels/knxnet_ip/badges/master/pipeline.svg)](https://gitlab.uptime.dk/troels/knxnet_ip/commits/master) [![coverage report](https://gitlab.uptime.dk/troels/knxnet_ip/badges/master/coverage.svg)](https://gitlab.uptime.dk/troels/knxnet_ip/commits/master)
+# KNXnetIP [![CircleCI](https://circleci.com/gh/uptimedk/knxnet_ip.svg?style=svg)](https://circleci.com/gh/uptimedk/knxnet_ip) [![codecov](https://codecov.io/gh/uptimedk/knxnet_ip/branch/master/graph/badge.svg)](https://codecov.io/gh/uptimedk/knxnet_ip)
```

Comment:
If you use `?style=shield` it will look more like the codecov badge
```suggestion
# KNXnetIP [![CircleCI](https://circleci.com/gh/uptimedk/knxnet_ip.svg?style=shield)](https://circleci.com/gh/uptimedk/knxnet_ip) [![codecov](https://codecov.io/gh/uptimedk/knxnet_ip/branch/master/graph/badge.svg)](https://codecov.io/gh/uptimedk/knxnet_ip)
```